src/overlays/actors/ovl_Bg_Dblue_Balance/z_bg_dblue_balance.h

```diff
@@ -6,12 +6,43 @@
 struct BgDblueBalance;
 
 typedef void (*BgDblueBalanceActionFunc)(struct BgDblueBalance*, GlobalContext*);
+typedef void (*BgDblueBalanceUnkFunc)(Actor*, GlobalContext*);
+
+#define BGDBLUEBALANCE_GET_7F(thisx) ((thisx)->params & 0x7F)
+#define BGDBLUEBALANCE_GET_300(thisx) (((thisx)->params >> 8) & 3)
+
+typedef struct {
+    /* 0x00 */ Vec3f unk_00;
+    /* 0x0C */ s16 unk_0C;
+    /* 0x0E */ u8 unk_0E;
+    /* 0x10 */ f32 unk_10;
+} BgDblueBalanceStruct; // size = 0x14
 
 typedef struct BgDblueBalance {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x24];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x015C */ Actor* unk_15C;
+    /* 0x0160 */ Actor* unk_160;
+    /* 0x0164 */ Actor* unk_164;
```

Comment:
```suggestion
    /* 0x015C */ struct BgDblueBalance* unk_15C;
    /* 0x0160 */ struct BgDblueBalance* unk_160;
    /* 0x0164 */ struct BgDblueBalance* unk_164;
```
Looks like these are always `BgDblueBalance*` and access fields from it. Any reason not to just have the members be of that type?